fn/templates/fn-deployment.yaml

```diff
@@ -58,9 +58,9 @@ spec:
             value: "api"
           - name: FN_PUBLIC_LB_URL
             {{- if .Values.tls.enabled }}
-            value: https://{{ .Release.Name }}.{{ .Values.fn_lb_runner.service.ingress_hostname }}:{{ .Values.fn_lb_runner.service.port }}
+            value: https://{{ template "fn.public_lb_url" . }}
```

Comment:
that's too complicated and hard to search for references,
here's what you need to do:
```yaml
- name: FN_PUBLIC_LB_SCHEMA
{{- if .Values.tls.enabled }}
  value: https
{{ else }}
  value: http
{{- end }}
- name: FN_PUBLIC_LB_URL:
  value: $(FN_PUBLIC_LB_SCHEMA)://{{ .Release.Name }}.{{ .Values.fn_lb_runner.service.ingress_hostname }}:{{ .Values.fn_lb_runner.service.port }}
```
that's it, looks similar to what we have for FN_DB_URL (clear and simple)

FYI, templates are for more complex logic, but not for the one-line templates.

Reply:
you know, actually, ignore my comment, these two fixes doing the same but in a different manner, which is totally fine.